crates/typst-library/src/foundations/label.rs

```diff
@@ -1,7 +1,10 @@
 use ecow::{eco_format, EcoString};
 use typst_utils::{PicoStr, ResolvedPicoStr};
 
-use crate::foundations::{func, scope, ty, Repr, Str};
+use crate::{
+    diag::StrResult,
+    foundations::{bail, func, scope, ty, Repr, Str},
+};
```

Comment:
```suggestion
use crate::{
    diag::StrResult,
    foundations::{bail, func, scope, ty, Repr, Str},
};
```
please ungroup these imports